app/src/common/constants.ts

```diff
@@ -38,8 +56,42 @@ export const GRAPH_RINKEBY_HTTP =
 export const GRAPH_RINKEBY_WS =
   process.env.REACT_APP_GRAPH_RINKEBY_WS || 'wss://api.thegraph.com/subgraphs/name/gnosis/omen-rinkeby'
 
-// Corona version options
-export const IS_CORONA_VERSION = process.env.REACT_APP_IS_CORONA_VERSION === 'true'
+export const USE_3BOX = !IS_CORONA_VERSION
+export const USE_DISQUS = IS_CORONA_VERSION
```

Comment:
Since we won't have both comments at the same time, maybe we should have only one variable here, like `USE_DISQUS`, and use 3box when it's false.